README.md

```diff
@@ -9,17 +9,11 @@
 Note, the name of the package available on PyPi is `casperlabs-client` (with hyphen),
 but the name of the library as well as the CLI is written with underscore: `casperlabs_client`.
 
-## Installation
+## Linux Installation
 
-`casperlabs-client` is a Python 3.7+ module, it does not support Python 2.7.
-
-Note: we highly recommend using
-[pipenv](https://github.com/pypa/pipenv)
-or
-[virtualenv](https://virtualenv.pypa.io/en/latest/).
-
-### Linux
+These instructions are for UBuntu 18.04 (or greater)
```

Comment:
i'd stick to 18.04 right now as I haven't verified this on anything else.

Reply:
Many of us are running 20.04, but with Python 3.8 (which is default).

Reply:
```suggestion
These instructions are for Ubuntu 18.04 and 20.04.  

Note that default versions of Python you will use is different for both versions of Ubuntu.
```